### PR TITLE
Initial commit for GPS direction and positioning lab

### PR DESCRIPTION
**Pull request by moizhraj**
This commit contains a lab with multiple parts showing GPS Direction and Positioning using an STM32L475E-IOT01A board and a Neo 6M GPS sensor using Azure IoT services and Azure Maps.

### CODE REVIEW

**Comment by jimbobbennett** [comment]
Can you add a link to this to the README.md file in the labs folder please.

**Review by jimbobbennett — CHANGES_REQUESTED** [review]

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[submodule "labs/iot/gps-directions-and-positioning/code/stm32/core/lib/threadx"]
```
[the remark this answers is not in the document]
Comment (moizhraj) [reply .gitmodules:1]: Other way is to include the entire source.

If user is just doing a git clone then these submodules will not initialize and fetch.

File: labs/iot/gps-directions-and-positioning/README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+| Date | October 2020 |
+| Learning Objectives | <ul><li>Set up dev environment</li><li>Set up IoT Hub</li><li>Send data from a device to IoT Hub</li><li>Call a device function from IoT Hub/Azure IoT Explorer</li><li>Send Cloud to Device message from IoT Hub/Azure IoT Explorer</li><li>Consume data from default Event Hub endpoint</li><li>Visualize co-ordinates and direction on Azure Maps</li></ul> |
```
[the remark this answers is not in the document]
Comment (moizhraj) [reply labs/iot/gps-directions-and-positioning/README.md:15]: Not really, unless we want to include a separate document to explain on how to setup RTOS? Let me know.

[the remark this answers is not in the document]
Comment (jimbobbennett) [reply labs/iot/gps-directions-and-positioning/README.md:15]: But the user has to set this up to make the sample work, so doesn't that count as a learning objective?

**Pull request closed without merging**